File: exercises/beecrowd/1074/1074.go
// https://judge.beecrowd.com/pt/problems/view/1074
package main

import (
	"fmt"
)

func main() {
	var n, x int
	fmt.Scan(&n)

	for i := 0; i < n; i++ {
		fmt.Scan(&x)

		if x == 0 {
			fmt.Println("NULL")
		} else {
			if x%2 == 0 {
				fmt.Print("EVEN ")
			} else {
				fmt.Print("ODD ")
			}

			if x > 0 {
				fmt.Println("POSITIVE")
			} else {
				fmt.Println("NEGATIVE")
			}
		}
	}
}

File: exercises/beecrowd/1074/1074.java
// https://judge.beecrowd.com/pt/problems/view/1074
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int n = sc.nextInt();

        for (int i = 0; i < n; i++) {
            int v = sc.nextInt();

            if (v == 0) {
                System.out.println("NULL");
            } else {
                String tipo = (v % 2 == 0) ? "EVEN" : "ODD";
                String sinal = (v > 0) ? "POSITIVE" : "NEGATIVE";
                System.out.println(tipo + " " + sinal);
            }
        }
    }
}
